src/main/java/com/example/PascalTriangle.java

```diff
@@ -2,10 +2,28 @@
 
 public class  PascalTriangle{
   
+private static int[] rowCalc(int[] array, int actualRow, int rowNum) {
```

Comment:
Indentation looks a bit off to me. Could you fix that?

Reply:
I don't like VSCode for this one. Somehow, I can not follow intendation errors visually as easily as I do in IntelliJ. Corrected, Willing to push a new version soon.